report_templates/subscriptions.erb

```diff
@@ -0,0 +1,32 @@
+<%#
+name: Subscriptions
+snippet: false
+template_inputs:
+- name: Subscriptions filter
+  required: false
+  input_type: user
+  description: Search query to filter subscription entitlements. E.g. `quantity >
+    0` to exclude subscriptions with unlimited entitlements. Keep unset for all subscriptions.
+  advanced: false
+- name: Output format
```

Comment:
Is yaml formatter stable enough already? I'm not sure I'd expose this as an input, once we think it's ready, we should have that as separate parameter for generating the report (as long as all reports will always support it). Overridding it per template using user input seems like something we'd need to remove in future.